elfpy/pricing_models/yieldspace.py

```diff
@@ -396,23 +385,12 @@ def calc_in_given_out(
                 )
                 * share_price  # convert to base
             )
-            # The fees are calculated as the difference between the bonds
-            # received and the base paid without slippage times the fee
-            # percentage. This can also be expressed as:
-            #
-            # fee = (1 - p) * phi * d_y
-            fee = (1 - spot_price) * trade_fee_percent * d_bonds
-            logging.debug(
-                ("fee = (1 - spot_price) * _fee_percent * d_bonds = (1 - %g) * %g * %g = %g"),
-                spot_price,
-                trade_fee_percent,
-                d_bonds,
-                fee,
-            )
+            curve_fee = abs(d_bonds - without_fee_or_slippage) * trade_fee_percent
```

Comment:
hy do you need abs?

Reply:
fixed in 36ec7f8093dd976989f9f9c157880796a8781a0d